Audio/AudioEngine.cpp

```diff
@@ -490,9 +437,6 @@ HRESULT AudioEngine::Impl::Reset(const WAVEFORMATEX* wfx, const wchar_t* deviceI
     //
     // Create mastering voice for device
     //
-
-#if defined(USING_XAUDIO2_8) || defined(USING_XAUDIO2_9)
```

Comment:
XAudio 2.8 and XAudio 2.9 are much closer in their API than XAudio 2.7, which removed a lot of alt codepaths